tests/test_ld.py

```diff
@@ -26,15 +26,18 @@
 import distro
```

Comment:
#123 implements a whole refactoring of the test env. mind basing your tests on that branch?